### DISCUSSION TITLE
how to connect on uploaded contract

### DISCUSSION THREAD

**Discussion by gunomnifluencer** [smartcontractkit/full-blockchain-solidity-course-py#584] [Q&A]
Hello,

In **remix** when we deployed contract it always shown contracts on the bottom left side,

how to display and connect contracts like on remix?

please answer

Thanks

**Comment by cromewar** [comment]
Hello @gunomnifluencer I don't know if I understood the question, but if I'm not wrong you are asking how could we deploy the contracts as we do on remix in other editors like vscode, right? 
Well that's not possible, that section of the contracts on Remix is an additional layer (made on the frontend) to show the contract and is characteristics, and the other frameworks and tools does not have that functionality.

**Comment by ZeroEkkusu** [comment]
If you meant how to load a contract you've deployed earlier (because you want to interact with it using the Remix UI), your compiled code and the network you're connected to must be the same as when you did the deployment. Then, just paste the contract's address in the `Load contract from Address` field in the `DEPLOY & RUN TRANSACTIONS` tab, and load it.
